src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1076,6 +1076,13 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('collation',
                    help='The collation of the managed instance.')
 
+        c.argument('proxy_override',
+                   arg_type=get_enum_type(ServerConnectionType),
+                   help='The connection type used for connecting to the instance.')
+
+        c.argument('public_data_endpoint_enabled',
+                   help='Whether or not the public data endpoint is enabled for the instance.')
```

Comment:
You can add `arg_type=get_three_state_flag()` here so it will be treated like `SwitchParameter` in powershell

Reply:
Great. Thanks